config.yml

```
@@ -1880,19 +1880,55 @@ nodes:
     fields:
       - name: index
         type: node
+        comment: |
+          The index variable for `for` loops.
```

```suggestion
          The index expression for `for` loops.
```

This can be quite a few different kinds of expressions. This includes `for foo`, but also `for foo, bar, baz`, `for @foo`, `for $foo`, etc. Could you put a little more into this message explaining that?